### ISSUE TITLE
[Bug] I am getting "Mono.Linker.MarkException" error on Adroid Release Mode

### ISSUE THREAD

**Issue by kerberosargos** [xamarin/XamarinCommunityToolkit#1644]
**My release mode setting as below**

![image](https://user-images.githubusercontent.com/11364304/135852897-b33f4570-fe6f-46c5-9022-914b6f8ad3f6.png)

**My LinkerSettings.xml**
```
<?xml version="1.0" encoding="utf-8" ?>
<linker>
	<assembly fullname="Xamarin.CommunityToolkit" />
</linker>

```

**Error**
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Mono.Linker.MarkException: Error processing method: 'System.Void Xamarin.CommunityToolkit.Android.Effects.SemanticEffectRouter/SemanticAccessibilityDelegate::OnInitializeAccessibilityNodeInfo(Android.Views.View,AndroidX.Core.View.Accessibiity.AccessibilityNodeInfoCompat)' in assembly: 'Xamarin.CommunityToolkit.dll' ---> Mono.Cecil.ResolutionException: Failed to resolve AndroidX.Core.View.Accessibiity.AccessibilityNodeInfoCompat
   at Mono.Linker.Steps.MarkStep.HandleUnresolvedType(TypeReference reference)
   at Mono.Linker.Steps.MarkStep.MarkType(TypeReference reference)
   at MonoDroid.Tuner.MonoDroidMarkStep.MarkType(TypeReference reference)
   at Mono.Linker.Steps.MarkStep.ProcessMethod(MethodDefinition method)
   at Mono.Linker.Steps.MarkStep.ProcessQueue()
   --- End of inner exception stack trace ---
   at Mono.Linker.Steps.MarkStep.ProcessQueue()
   at Mono.Linker.Steps.MarkStep.ProcessPrimaryQueue()
   at Mono.Linker.Steps.MarkStep.Process()
   at Mono.Linker.Steps.MarkStep.Process(LinkContext context)
   at MonoDroid.Tuner.MonoDroidMarkStep.Process(LinkContext context)
   at Mono.Linker.Pipeline.ProcessStep(LinkContext context, IStep step)
   at Mono.Linker.Pipeline.Process(LinkContext context)
   at MonoDroid.Tuner.Linker.Process(LinkerOptions options, ILogger logger, LinkContext& context)
   at Xamarin.Android.Tasks.LinkAssemblies.Execute(DirectoryAssemblyResolver res)
   at Xamarin.Android.Tasks.LinkAssemblies.RunTask()
   at Microsoft.Android.Build.Tasks.AndroidTask.Execute() in /Users/builder/azdo/_work/1/s/xamarin-android/external/xamarin-android-tools/src/Microsoft.Android.Build.BaseTasks/AndroidTask.cs:line 17	Mors.Android			

```

**Comment by jfversluis** [comment]
Any reason you're ignoring our bug template? :) Now I'm missing some valuable information like which version of Forms you are using or which version of the Toolkit.

Could you add that? Thanks!

**Comment by kerberosargos** [comment]
Hi @jfversluis

**Comment by PureWeen** [comment]
@kerberosargos This is something that was fixed in version 1.3.2.2 of AndroidX.Core and 1.1.0.7 of Recycler View

You most likely need to clean your obj/bin or see if any of your projects are referencing AndroidX libraries explicitly 

You can also try explicitly referencing the latest version of this https://www.nuget.org/packages/Xamarin.AndroidX.Core

I'm going to close this for now. If you are still having the issue and can attach a reproduction we'll reopen

**Comment by kerberosargos** [comment]
Hello @PureWeen and @jfversluis  again 

My whole confugarion as below

![image](https://user-images.githubusercontent.com/11364304/135976383-191b08a1-b4a6-49c1-9ed1-ea06c75dba12.png)

![image](https://user-images.githubusercontent.com/11364304/135976503-2fd7ac96-2149-46db-a7bd-ea22b4b2fd59.png)

![image](https://user-images.githubusercontent.com/11364304/135976560-5e884b0a-776f-4e54-88c4-7acd60d73b19.png)

![image](https://user-images.githubusercontent.com/11364304/135976653-a53e3cbf-73cc-4c2f-a3f3-f57918ae25ad.png)

**I am getting same error**

![image](https://user-images.githubusercontent.com/11364304/135976712-d01f74da-713c-41d1-a88d-e35aea19d29c.png)
